octodns/provider/azuredns.py

```diff
@@ -898,6 +905,22 @@ def _data_for_dynamic(self, azrecord):
 
         return data
 
+    def _process_desired_zone(self, desired):
+        # check for support of force up/down values
+        for record in desired.records:
+            if not getattr(record, 'dynamic', False):
+                continue
+            for name, pool in record.dynamic.pools.items():
+                for value in pool.data['values']:
+                    if value['up']:
+                        # Azure only supports up=None & up=False, not up=True
+                        msg = f'up=True is not supported for "{name}" pool ' \
+                            f'in {record.fqdn}'
+                        fallback = 'ignoring it'
+                        self.supports_warn_or_except(msg, fallback)
```

Comment:
Will the `ignoring it` here result in persistent changes or will it just be set and ignored by azure when created and not seen as a diff when planning changes? If it will show as a diff we'll probably want to do a modification here like with `Route53Provider` even if it results in a setup that doesn't make sense.

Reply:
Although no supported, the provider is marking in the endpoint name that the pool value is set to up=True so that we can parse the flag correctly in the plan phase and not show unnecessary changes. I'd rather no drop the up flag here since Azure does support up=False.

Reply:
I now got what you meant, and it makes sense to me to reset the flag when there's an unsupported case (`up=True`). I've added that change to the PR.